cpp/src/gandiva/gdv_function_stubs.cc

```diff
@@ -892,6 +892,28 @@ const char* gdv_fn_aes_decrypt(int64_t context, const char* data, int32_t data_l
 
   return ret;
 }
+
+// Expand inner macro for all integer types.
+#define NEGATIVE_FN_INTEGER_PARAMS(INNER) \
```

Comment:
why not include them in the precompiled code

Reply:
@projjal because I use the `std::numeric_limits<gdv_##TYPE>::min()` function to avoid overflow and if I remember, we can not use functions from stdlib in precompiled code

Reply:
You can use limit macros defined in stdint.h

Reply:
I forget about it, I will apply this change now